drizzlepac/haputils/astrometric_utils.py

```diff
@@ -46,16 +46,18 @@
 from astropy.utils.decorators import deprecated
 
 import photutils  # needed to check version
-if Version(photutils.__version__) < Version('1.1.0'):
+if Version(photutils.__version__) < Version('1.5.0'):
     OLD_PHOTUTILS = True
-    from photutils.segmentation import (detect_sources,
-                                        deblend_sources, make_source_mask)
-    from photutils.segmentation import source_properties as SourceCatalog
+    from photutils.segmentation import (detect_sources, SourceCatalog,
```

Comment:
This may be trivial, but would it make this code a little easier to follow if the imports common to both the old and new versions were pulled out of the if statement.  You could have a statement like:

from photutils.segmentation import (detect_sources, deblend_sources, SegmentationImage) 

come before you even call the if statement, and then the imports inside the if statement would only import functions/classes specific to that version of the library.  Not critical to the code running, but more a maintenance issue perhaps.

Reply:
Made the suggested change!